plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/GcsDataflowProjectClient.java

```diff
@@ -88,13 +89,16 @@ public StagingLocationVerificationResult createStagingLocation(
     }
   }
 
-  private static String toGcsBucketName(String stagingLocation) {
+  public static String toGcsBucketName(String stagingLocation) {
     String gcsLocation;
     if (stagingLocation.startsWith(GCS_PREFIX)) {
```

Comment:
Elsewhere code that now calls this method tested the prefix using case-insensitivity.